versions.yaml

```diff
@@ -126,7 +126,7 @@ assets:
         name: "ubuntu"
         version: "latest"
       x86_64:
-        name: &default-image-name "clearlinux"
+        name: &default-image-name "ubuntu"
```

Comment:
I looked at https://github.com/kata-containers/kata-containers/pull/2908 but didn't see mention of clearlinux there? The rationale to switch to Ubuntu seems to be to make building golang apps easier but clearlinux tends to be updated daily (sometimes more so) so I'd be surprised if it didn't come with the latest possible version of golang ;) It's also smaller than Ubuntu.

Reply:
Ah. I've removed it again because this was only for image (not initrd, and only initrd will work ATM) anyway.
To answer the question still: We use Fedora, not Clear Linux, as a build container for Clear Linux. (On architectures that cannot use musl with Rust, this could lead to glibc mismatches, but Clear Linux is not supported on those anyway.) I excluded Clear Linux from #2908 because it would require an update to the Fedora build container and historically, we experienced too much pain on those upgrades (cf. #2128, #2150) to make it worthwile for a solution that is temporary anyhow for we will move to fully rustified image pulls (cf. #3035).